### PR TITLE
Global styling

### PR DESCRIPTION
**Pull request by baileytalks**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,41 @@
...
  */
+
+body {
+  background-color:  white;
+}
+
+/* We need a global font, h1, h2, h3, p, navbar colour, footer colour, body colour
+ * button colours
+ */
+
```
Comment (adamjohnsnow) [inline app/assets/stylesheets/application.css]: Are these comments still relevant?

**Review by adamjohnsnow — APPROVED** [review]
Only some vestigial comments to clean up, but approved!!

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,41 @@
  *= require_tree .
  *= require_self
  */
+
+body {
```
Comment (neoeno) [inline app/assets/stylesheets/application.css]: You might have a nicer time if you can extract this to something like `base.css` — and make `application.css` only responsible for being the place other stuff is required. Not a blocker for this though, as it does seem to be genuinely global styling.

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (neoeno) [reply app/assets/stylesheets/application.css]: Are they?

**Review by neoeno — APPROVED** [review]
Great work! Just a few comments, not requirements for changes. Merge when you're ready.

**Pull request merged**